src/systems/filecoin_blockchain/storage_power_consensus/storage_power_actor.md

```diff
@@ -14,58 +14,69 @@ title: Storage Power Actor
 
 {{< readfile file="/docs/actors/builtin/storage_power/storage_power_actor_code.go" code="true" lang="go" >}}
 
+{{<label storage_power>}}
+## On Power
```

Comment:
why the move to here? I feel like it is more general than just the actor...

Reply:
moved there because thats where power was introduced but when I was linking the docs, think it makes sense to exist at where it was. It has been moved back.